packages/client-core/src/admin/common/hooks/Instance.hooks.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from 'react'
+
+export const useFetchAdminInstance = (user, adminInstanceState, InstanceService) => {
+  useEffect(() => {
+    if (user?.id.value != null && adminInstanceState.updateNeeded.value === true) {
```

Comment:
`user?.id.value && adminInstanceState.updateNeeded.value` would be same as above